modules/xmpp/TraceablePeerConnection.js

```diff
@@ -392,6 +392,13 @@ TraceablePeerConnection.prototype.setRemoteDescription
     description.sdp = this.rtxModifier.implodeRemoteRtxSsrcs(description.sdp);
     this.trace('setRemoteDescription::postTransform (implodeRemoteRtxSsrcs)', dumpSDP(description));
 
+    if (this.session.room.options.preferH264) {
```

Comment:
@hristoterezov will do, but i'm having a hard time determining which section of the doc it should go in.  do you know?